service/controller/v1/prometheus/spec.go

```diff
@@ -1,12 +1,12 @@
 package prometheus
 
 const (
-	// prometheusConfigPath is the Prometheus route that returns the current
-	// configuration webpage.
-	prometheusConfigPath = "/config"
-	// prometheusReloadPath is the Prometheus API route that reloads the configuration
+	// ConfigPath is the Prometheus route that returns the current
+	// configuration.
+	ConfigPath = "/api/v1/status/config"
```

Comment:
I exposed these since that information is needed in `ConfigMap` resource implementation's unit tests as well.

Reply:
I'd rather copy the string in the test. That allows to:

- Prevent typos to some degree.
- Make test more explicit.
- Keep that private.

Reply:
It was previously copied and when it changed, it wasted quite some unnecessary time because the test failure was far from obvious due to this. I don't think this affects test explicitness at all because the only place where this is used is "home made" router within handler in test.

If you still think I should revert this to hidden, I can do it.

Reply:
I'm not supper fussed. This isn't a library project. 👍